hr_holidays_accrual_advanced/wizards/hr_leave_allocation_accrual_calculator.py

```diff
@@ -83,17 +75,14 @@ def _recalculate(self):
         )
 
         balance = 0.0
-        accruement_ids = [(5, False, False)]
         for accruement in accruements:
             balance += accruement.days_accrued
-            accruement_ids.append((0, 0, {
+            accruement_ids |= CalculatorAccruement.new({
                 'days_accrued': accruement.days_accrued,
                 'accrued_on': accruement.accrued_on,
                 'reason': accruement.reason,
-            }))
+            })
 
-        self.write({
-            'accrued': accrued,
-            'balance': balance,
-            'accruement_ids': accruement_ids,
-        })
+        self.accrued = accrued
```

Comment:
Why not writing in block?

Reply:
You can not write many2many relation on non-saved record without crash

Reply:
`accrued` are records? What a bad variable nomenclature then...